plane/source/docs/quadplane-vtol-tuning-process.rst

```diff
@@ -217,30 +217,50 @@ This test will allow to test the altitude controller and ensure the stability of
 
  If the QuadPlane in QHOVER starts to move up and down, the vertical position and velocity controllers may need to be reduced by 50%. These values are: :ref:`Q_P_POSZ_P <Q_P_POSZ_P>` and :ref:`Q_P_VELZ_P <Q_P_POSZ_P>`.
 
+.. note:: If the :ref:`Q_M_THST_HOVER<Q_M_THST_HOVER>` learned should be ~0.3-0.6. Higher values indicate that insufficient thrust is available, either due to motor system design, obstructed prop air flow by the fuselage or wings, or excessive yaw bias (see next section)
+
 Step 8: Yaw Bias
 ----------------
 
 A common problem in QuadPlanes is excessive amount of VTOL power being
-used to maintain yaw. This can be caused by:
+used to maintain yaw hold intead of providing lift. This can be caused by:
 
 - small misalignment of the VTOL motors
 - frame twist (often caused by wing twist) as thrust is applied
+- obstructed prop air flow by wings in TVBS type tailsitters making vectored yaw by motor tilt less effective. This can also cause enough reduction in normal thrust to make hovering and VTOL climb difficult.
```

Comment:
Could you link to a definition of "TVBS" somewhere? I didn't know what this was.